### ISSUE TITLE
"No data present for Zip64Extended info" occured when extracting

### ISSUE THREAD

**Issue by gomsang** [srikanth-lingala/zip4j#129]
First, thank you for providing a good library.

I have a question while using the library.

The following error occurs while extracting several compressed files. "No data present for Zip64Extended info"

I have attached a compressed file that causes an error. I would appreciate your reply.

[Houses under the sea(3eda32b9.0).zip](https://github.com/srikanth-lingala/zip4j/files/4080435/Houses.under.the.sea.3eda32b9.0.zip)

**Comment by srikanth-lingala** [comment]
This is an issue with the tool that generated the zip file. As the error message says, the headers in the zip file specify that there is a zip64 info present in the header section, but there is no data under that header.

However, I have modified the code in zip4j to not throw an exception, and simply ignore if there is no data present. I have already changed the code . I will include this code change in the next release.
